ChangeLog.md

```diff
@@ -4,9 +4,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Upcoming: Changes for freud 2.0
 
+### Added
+* Ability to specify NeighborQuery objects as points for neighbor-based pair computes.
+* Various validation tests.
+
+### Changed
+* All compute objects that perform neighbor computations now use NeighborQuery internally.
+* All compute objects that perform neighbor computations now loop over NeighborBond objects.
+* Renamed ref\_points, points to points, query\_points to clarify their usage.
+* Standardized naming of various standard parameters across freud.
```

Comment:
![image](https://user-images.githubusercontent.com/3943761/62886159-993fe400-bd08-11e9-8a30-3ce7c0bcde42.png)

Reply:
What's redundant? The second line is referring to things like rcut, num_neighbors, etc. Could make that explicit if you like.

Reply:
Sorry, I didn't make my comment clear; "Standardized ... standard parameters" sounded redundant.

Reply:
Got it.